database/sql/V011__extend_orders_for_contract_upgrade.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TYPE BalanceFrom AS ENUM ('owner', 'vault_internal', 'vault_external');
+CREATE TYPE BalanceTo AS ENUM ('owner', 'vault_internal');
```

Comment:
nit: These should probably be the same as the SC enum:
```suggestion
CREATE TYPE BalanceFrom AS ENUM ('erc20', 'external', 'internal');
CREATE TYPE BalanceTo AS ENUM ('erc20', 'internal');
```

Reply:
I will do this, but I don't particularly like that internal and external make no reference to the fact that these are vault related.

Reply:
I started a discussion about this here: https://github.com/gnosis/gp-v2-contracts/issues/543

We can always re-release a new version with just name changes to make it clearer and then update it here to match the new names. However, I think it makes sense for the enum values to match the values that get signed with the order.